src/pages/komodo-defi-framework/api/common_structures/wallet/index.mdx

```diff
@@ -62,3 +67,47 @@ The `InputTxns` object includes the following items:
     }
   ```
 </CollapsibleSection>
+
+### PayForGas
+
+The `PayForGas` object includes the following items:
+
+| Parameter                    | Type    | Description                                                                                                                                                                  |
+| ---------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| tx\_type                     | string  | ERC20-like coins and tokens only. Options are `Legacy` or `Eip1559`. The type of transaction values being configured.                                                        |
```

Comment:
Not only ERC20
maybe 'ETH-like' (or EVM-like if 'EVM' is well known)

Reply:
fixed, thanks